### PR TITLE
Check embedded code snippets are up-to-date

### PR DESCRIPTION
**Pull request by daviddrysdale**
Regenerate with `embedmd -w docs/<file>.md`

### CODE REVIEW

**Comment by daviddrysdale** [comment]
This is a commit from #247, pulled out separately to see if it can squeeze past Travis timeout limits on its own.

File: Dockerfile [hunk 1]
```diff
@@ -48,3 +48,12 @@ RUN rustup component add \
   rls \
   rust-analysis \
   rust-src
+
+# Install embedmd (via Go)
+ARG GOLANG_VERSION=1.13.1
+ENV GOROOT /usr/local/go
+ENV GOPATH $HOME/go
+ENV PATH $GOROOT/bin:$GOPATH/bin:$PATH
+RUN mkdir -p ${GOROOT}
```
Comment (tiziano88) [inline Dockerfile]: (nit) We seem to not put variables in {} in this file, unless necessary (and below)

```suggestion
RUN mkdir -p $GOROOT
```

File: scripts/check_formatting [hunk 2]
```diff
@@ -20,3 +20,8 @@ find scripts -type f -exec shellcheck {} +
 # Fortunately, rustfmt has the --check option that will make it exit with 1
 # if formatting has to be applied.
 find examples rust -type f -name '*.rs' -exec rustfmt --check {} +
+
+# Check embedded code snippets are up-to-date.
+grep --recursive --files-with-matches embedmd docs | while IFS= read -r file; do
```
Comment (tiziano88) [inline scripts/check_formatting]: Also apply the corresponding fix in the `format` script?

Comment (daviddrysdale) [reply scripts/check_formatting]: Ah, I hadn't noticed there was a `format` script &ndash; done.

File: Dockerfile (hunk 1 above)
Comment (daviddrysdale) [reply Dockerfile]: Done.

**Pull request merged**